src/sentry/static/sentry/app/utils.jsx

```diff
@@ -232,6 +234,18 @@ export function getShortVersion(version) {
   return version;
 }
 
+export function parseBitBucketRepo(repo) {
+  if (repo) {
+    let re = /bitbucket\.org\/([^\/]+\/[^\/]+)/i;
+    let match = repo.match(re);
+    let parsedRepo;
+    match ? (parsedRepo = match[1]) : (parsedRepo = repo);
```

Comment:
ik you were just matching the other form, but this should rly be
` let parsedRepo =  match ? match[1] : repo;`
imo